### DISCUSSION TITLE
There is no info about database in installation documentation and CLI

### DISCUSSION THREAD

**Discussion by teinett** [strapi/strapi#6591] [Legacy Topics]
I am trying to install Strapi 3.0.1.

**Problem**: as a developer I would like to know the type of installed database, but there is no such information.

**Docs**: https://strapi.io/documentation/v3.x/getting-started/quick-start.html
There is no information about default database in docs.

**Docs**: https://strapi.io/documentation/v3.x/installation/cli.html
I could find only info that it's possible to use another database than default one. But there is no information which one is default database. 

**Installation process with CLI**: `yarn create strapi-app my-project --quickstart`
There is no information about default database in help text during installation process.

Node: 12.18.0
Yarn: 1.22.4

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Comment by derrickmehaffy** [comment]
Please see the note just above this line in the tutorial: https://strapi.io/documentation/v3.x/getting-started/quick-start-tutorial.html#_2-create-an-administrator-and-front-end-user

![image](https://user-images.githubusercontent.com/8593673/84075242-a9583c80-a988-11ea-83c5-2ee901422ef5.png)

**Comment by teinett** [comment]
@derrickmehaffy  I mentioned specifically these 3 places (two doc pages and the CLI installation process). 

I stuck for 1h+ for the moment, because all the info about options of installation is in several places. I installed Strapi and deleted the folder for several times. It's "try and get not expected" process. I am really sad about it. 

I tried Strapi at @beta stage, and CLI asked me about the database during the process. I remember it: I will be asked in CLI. 
Now CLI doesn't ask about database. CLI does some magic and doesn't say what exactly.

So I give you my feedback: as a developer, I would like to know the very base info in these sources too: quick start pages and CLI installation process. It would be really helpful for beginners.

**Reply by derrickmehaffy** [reply]
Thank you for the feedback, there is a lot of planned changes to the docs and I'll pass these along to the Strapi team.
